src/routes/+page.svelte

```
@@ -459,6 +459,12 @@ Info flow: User evidence -> MeechieStudioTextSeam -> page spec -> image/package/
 		return '';
 	});
 
+	onDestroy(() => {
+		if (draftTimer) {
+			clearTimeout(draftTimer);
```

ESLint

[error] 'clearTimeout' is not defined.

(no-undef)